### ISSUE TITLE
tidal midi error for blofeld

### ISSUE THREAD

**Issue by guillaumeazer** [tidalcycles/Tidal#312]
Hello, I'm on windows and I installed tidal midi with stack, cabal didn't work.
in atom is working I can send midi notes from my computer to the blofeld but when I try to evaluate this line: import Sound.Tidal.Blofeld (to have access to parameters of the blofeld)
I have this error : : error:tidal> Could not find module `Sound.Tidal.Blofeld' Perhaps you meant Sound.Tidal.Bjorklund (from tidal-0.9.6) Sound.Tidal.MIDI.Blofeld (from tidal-midi-0.9.5.2)

 do you have an idea? thanks :)

Guillaume

**Comment by guillaumeazer** [comment]
okay, I don't know how but finally, cabal install tidal-midi worked, so I have in my cabal folder tidal-midi (ver 0.9.10). I can send midi notes via synthcontroller.
But it still don't find the module `Sound.Tidal.Blofeld'. And I don't know why because when I go to the tidal-midi\0.9.10 folder, I can see in the .tar there is a "blofeld.hs".

So why atom don't find the Sound.Tidal.Blofeld?

**Comment by guillaumeazer** [comment]
YES!!! Now it works. I saw some errors in the example for the blofeld. The right command to evaluate is import "Sound.Tidal.MIDI.Blofeld" and not "import Sound.Tidal.Blofeld". Also a lot of params have not the same name as in the documentation (https://github.com/tidalcycles/tidal-midi/blob/master/doc/blofeld-params.md). Maybe an update could be done? If I found a way to know exactly all param names I will send the list. :)
